rpc_util.go

```diff
@@ -164,7 +164,7 @@ func encode(c Codec, msg interface{}, pf payloadFormat) ([]byte, error) {
 	// Write message into the fixed header.
 	buf.WriteByte(uint8(pf))
 	var b []byte
-	var length uint32
+	var length uint
```

Comment:
len(...) returns int so that there is no overflow issue actually when length is int type on both 32 and 64 bit machines. Simply there is a compiling error when go 1.3 is used for the line:
if length > math.MaxUint32 

I guess Go 1.3 tries to convert math.MaxUint32 to int but the later Go version does not.